package-lock.json

```diff
@@ -0,0 +1,219 @@
+{
```

Comment:
@pvannierop we should get rid of this package.lock since we use yarn (this is npm)

Reply:
Updated.